python/pyspark/testing/utils.py

```diff
@@ -243,6 +245,10 @@ def assertDataFrameEqual(df: DataFrame, expected: DataFrame, check_row_order: bo
         If set to `False` (default), the row order is not taken into account.
         If set to `True`, the order of rows is important and will be checked during comparison.
 
```

Comment:
I think you can remove whitespaces between each parameter (https://numpydoc.readthedocs.io/en/latest/format.html#parameters)